interview-questions.md

```diff
@@ -7,48 +7,48 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
+  Your answer: An instance variable is similar to state object key-value pairs in React. Instance variables that belong to a class in ruby. They are different from other variables in that they cannot be accessed outside of the class, but only through methods in the class.
```

Comment:
Cool! I would be hesitant to answer with a example regarding React when the question is designated as Ruby. I know its easy to want to use a reference like this here, in this assessment, with us, as we have been working with React. 

But,  think of it this way, if you're being asked this by an interviewer at XYZ company out in the wild, the person asking this may not have any experience with the other technology you're trying to reference to and your example would be lost on them.